src/components/input/input.css

```diff
@@ -155,3 +155,23 @@ input:checked + .slider:before {
 input[name="radio"] {
 	accent-color: var(--COLOR-PRIMARY);
 }
+
+.input__error {
+	border: 2px solid var(--COLOR-DANGER);
+	outline: var(--COLOR-DANGER);
+}
+
+.input__error:hover {
+	border: 2px solid var(--COLOR-DANGER);
+	outline: var(--COLOR-DANGER);
```

Comment:
you can add fallback values to variables as well just in case that variable values is not supported please refer to:https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values

Reply:
will look into it